### PR TITLE
Adapter simplification and branch query addition

### PR DESCRIPTION
**Pull request by srvance**
This combines the contents of #70 and #71 into one PR to get around some nasty merge reconciliation that arose. Those two can be closed if this is accepted.

### CODE REVIEW

File: addon/adapters/github.js [hunk 1]
```diff
@@ -19,7 +19,8 @@ export default RESTAdapter.extend({
   }),
 
   pathForType(type) {
-    return Ember.String.camelize(Ember.String.pluralize(type.replace('github', '')));
+    return Ember.String.camelize(Ember.String.pluralize(type.replace('github', '')))
+      .replace('organizations', 'orgs');
```
Comment (srvance) [inline addon/adapters/github.js]: What do you think of just implementing `pathForType` in orgs to return a static string?

Comment (elwayman02) [inline addon/adapters/github.js]: I would prefer to try and be more future-proof by calculating it based on the actual type.

Comment (srvance) [inline addon/adapters/github.js]: Why don't I restore it the way it was until we can come up with something better? This is the effect of a design intention that can't play out completely.

**Comment by elwayman02** [comment]
Ok

**Comment by elwayman02** [comment]
LGTM

**Pull request merged**